docs/getting_started/install_ez_platform.md

```diff
@@ -1,12 +1,209 @@
+
+## Prepare work environment
+
+To install eZ Platform you need a stack with your operating system, MySQL and PHP.
+
+You can install it by following your favorite tutorial, for example: [Install LAMP stack on Ubuntu](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-ubuntu-18-04).
+
+[For production](#prepare-installation-for-production) you also need Apache as the HTTP server.
+[For development](#use-phps-built-in-server), you can use the built-in PHP server instead.
+
+You also need `git` for version control.
+
+Before getting started, make sure you review the [requirements](requirements.md) page to see the systems we support and use for testing.
+
+## Get Composer
+
+Install Composer, the PHP command line dependency manager.
+Use the package manager for your Linux distribution. For example on Ubuntu:
+
+``` bash
+apt-get install composer
+```
+
+!!! tip "Install Composer locally"
+
+    If you want to install Composer inside your project root directory only,
+    run the following command in the terminal:
+
+    ``` bash
+    php -r "readfile('https://getcomposer.org/installer');" | php
+    ```
+
+    After this you need to replace `composer` with `php -d memory_limit=-1 composer.phar` in all commands below.
+
+## Get eZ Platform
+
+There are two ways to get eZ Platform. The result is the same, so you can use the way you prefer:
+
+- [Download or clone](#a-download-ez-platform)
+- [Create a project with Composer](#b-create-project-with-composer)
+
+### A. Download eZ Platform
+
+You can download eZ Platform from two sources:
+
+1\. Download an archive
+
+- For open-source eZ Platform, download from [ezplatform.com](https://ezplatform.com/#download-option).
+- For licensed eZ Enterprise, download from the [Support portal](https://support.ez.no/Downloads).
+
+Extract the archive into the location where you want your project root directory to be.
+
+!!! enterprise "Authentication tokens for eZ Enterprise"
+
+    When installing eZ Enterprise, you need to [set up authentication tokens](install_ez_enterprise.md#setting-up-authentication-tokens-for-ez-enterprise).
+
+2\. Clone GitHub repository
+
+You can also clone the [GitHub repository](https://github.com/ezsystems/ezplatform).
+
+``` bash
+git clone https://github.com/ezsystems/ezplatform .
+```
+
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
+
+##### Install dependencies with Composer
+
+Composer will look inside the `composer.json` file and install all of the packages required to run eZ Platform.
+
+From the folder into which you downloaded eZ Platform, run:
+
+``` bash
+composer install
+```
+
+### B. Create project with Composer
+
+You can also use Composer to instantly create a project with all the dependencies:
+
+``` bash
+composer create-project --no-dev --keep-vcs ezsystems/ezplatform .
```

Comment:
Two comments regarding this command:
1. Maybe we should note that this will always install the newest eZ Platform, and you can specify version like this:
`composer create-project --no-dev --keep-vcs ezsystems/ezplatform . v2.2.2`
2. This command will create the project in the current directory (`.` in command) - this can be messy if someone just ctrl+c ctrl+v this into a terminal.

Reply:
Added detail about the installation folder and a tip about the version below